database/factories/MismatchFactory.php

```diff
@@ -46,6 +46,7 @@ public function definition()
                 }
                 return $this->getRandomValueForDatatype(self::PROPERTIES[$propertyId]);
             },
+            'meta_wikidata_value' => 'Q' . $this->faker->optional(0.7)->randomNumber(),
```

Comment:
I think this needs to depend on the `property_id` (like the `wikidata_value` above), otherwise it’ll generate meta values for non-time properties and cause validation failures. (I assume this is causing the CI errors.)